test/integration/edition_edit_test.rb

```diff
@@ -8,31 +8,90 @@ class EditionEditTest < IntegrationTest
     stub_linkables
   end
 
-  should "show document summary and title" do
-    edition = FactoryBot.create(:guide_edition, title: "Edit page title", state: "draft")
-    visit edition_path(edition)
-
-    assert page.has_title?("Edit page title")
-
-    row = find_all(".govuk-summary-list__row")
-    assert row[0].has_content?("Assigned to")
-    assert row[1].has_text?("Content type")
-    assert row[1].has_text?("Guide")
-    assert row[2].has_text?("Edition")
-    assert row[2].has_text?("1")
-    assert row[2].has_text?("Draft")
-  end
+  context "when edition is draft" do
+    setup do
+      edition = FactoryBot.create(:guide_edition, title: "Edit page title", state: "draft")
+      visit edition_path(edition)
+    end
+
+    should "show document summary and title" do
+      assert page.has_title?("Edit page title")
+
+      row = find_all(".govuk-summary-list__row")
+      assert row[0].has_content?("Assigned to")
+      assert row[1].has_text?("Content type")
+      assert row[1].has_text?("Guide")
+      assert row[2].has_text?("Edition")
+      assert row[2].has_text?("1")
+      assert row[2].has_text?("Draft")
+    end
+
+    should "show all the tabs for the edit" do
+      assert page.has_text?("Edit")
+      assert page.has_text?("Tagging")
+      assert page.has_text?("Metadata")
+      assert page.has_text?("History and notes")
+      assert page.has_text?("Admin")
+      assert page.has_text?("Related external links")
+      assert page.has_text?("Unpublish")
+    end
+
+    context "#metadata" do
```

Comment:
I missed this one last time round, but `metadata` is not the name of the method in this context, so would be better as something like "viewing the metadata tab".